src/services/clients/dissolutionApi.client.ts

```diff
@@ -67,6 +68,18 @@ export class DissolutionApiClient {
     return response.data
   }
 
+  public async patchDissolutionPaymentData(applicationReference: string, body: DissolutionPaymentPatchRequest): Promise<void> {
+    try {
+      return await this.axios.patch(
+        `${this.DISSOLUTIONS_API_URL}/dissolution-request/${applicationReference}/payment`,
+        body,
+        this.generateConfigForAPIKey()
+      )
+    } catch (err) {
+      return Promise.reject(err)
+    }
```

Comment:
this catch is pointless.

Reply:
Without it, and if the PATCH call fails, will the error handling middleware handle/catch it ok?

Reply:
Yup. This is the promise equivalent  of catch(err) { throw err }